src/Equinor.ProCoSys.IPO.Query/GetOutstandingIpos/OutstandingIposResultDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+
+namespace Equinor.ProCoSys.IPO.Query.GetOutstandingIpos
+{
+    public class OutstandingIposResultDto
+    {
+        public OutstandingIposResultDto(
+            int maxAvailable,
+            IEnumerable<OutstandingIpoDetailsDto> items)
+        {
+            MaxAvailable = maxAvailable;
```

Comment:
What is this for?
Isn't it always the length of Items

Reply:
Yes. It's just convenient for when we're going to use it in main since we're showing # of items and then listing them.

Reply:
Then I would have removed it from here. No value, and just causing you to create yet a dto here. Suggest that endpoint just delivers list of items